services/core-api/app/api/EMLI_contacts/models/EMLI_contact.py

```diff
@@ -0,0 +1,117 @@
+    is_general_contact = db.Column(db.Boolean, nullable=False, default=False)
+
+    emli_contact = db.relationship(
+        'EMLIContactType',
+        backref='emli_contact',
+        order_by='asc(EMLIContactType.display_order)',
+        lazy='joined')
+
+    @classmethod
+    def create(cls,
+               emli_contact_type_code,
+               mine_region_code,
+               email,
+               phone_number,
+               fax_number,
+               mailing_address_line_1,
+               mailing_address_line_2,
+               first_name,
+               last_name,
+               is_major_mine,
+               is_general_contact,
+               deleted_ind,
+               add_to_session=True):
+        new_contact = cls(
+            emli_contact_type_code=emli_contact_type_code,
+            mine_region_code=mine_region_code,
+            email=email,
+            phone_number=phone_number,
+            fax_number=fax_number,
+            mailing_address_line_1=mailing_address_line_1,
+            mailing_address_line_2=mailing_address_line_2,
+            first_name=first_name,
+            last_name=last_name,
+            is_major_mine=is_major_mine,
+            is_general_contact=is_general_contact,
+            deleted_ind=deleted_ind)
+        if add_to_session:
+            new_contact.save(commit=False)
+        return new_contact
+
+    @classmethod
+    def find_EMLI_contact(cls, emli_contact_type_code, mine_region_code=None, is_major_mine=None):
+        if is_major_mine and mine_region_code:
+            return cls.query.filter_by(
+                emli_contact_type_code=emli_contact_type_code,
+                mine_region_code=mine_region_code,
+                is_major_mine=is_major_mine).filter_by(deleted_ind=False).first()
+        elif mine_region_code:
+            return cls.query.filter_by(
+                emli_contact_type_code=emli_contact_type_code,
+                mine_region_code=mine_region_code).filter_by(deleted_ind=False).first()
+        return cls.query.filter_by(emli_contact_type_code=emli_contact_type_code).filter_by(
+            deleted_ind=False).first()
+
+    @classmethod
+    def find_EMLI_contact_by_id(cls, contact_id):
+        return cls.query.filter_by(contact_id=contact_id).filter_by(deleted_ind=False).first()
+
+    @classmethod
+    def find_EMLI_contacts_by_mine_region(cls, mine_region_code, is_major_mine=None):
+        general_contacts = cls.find_EMLI_general_contacts()
+        mmo_contact = cls.find_major_mine_office()
+
+        if is_major_mine == True:
+            return cls.query.filter_by(
+                mine_region_code=mine_region_code, is_major_mine=is_major_mine).filter_by(
+                    deleted_ind=False).union(mmo_contact).union(general_contacts).all()
+        elif is_major_mine == False:
+            return cls.query.filter_by(
+                mine_region_code=mine_region_code, is_major_mine=is_major_mine).filter_by(
```

Comment:
in both of these the deleted_ind filter can be merged into the previous one to remove the additional use of filter_by